code/addons/docs/package.json

```diff
@@ -36,11 +36,7 @@
       "require": "./dist/preview.js",
       "import": "./dist/preview.mjs"
     },
-    "./preset": {
-      "types": "./dist/preset.d.ts",
-      "require": "./dist/preset.js",
-      "import": "./dist/preset.js"
-    },
+    "./preset": "./dist/preset.js",
```

Comment:
are these changes (and on line 50) necessary? why?

Reply:
Yes, they are. `addon-docs` wasn't using the `addon-bundle` script, but the old build script instead. I couldn't build addon-docs with the old script, though, because I was using an import to `fs` and the build script assumed that I would build for the browser, which isn't the case for a Webpack5 loader. That's the reason why I have upgraded addon-docs to use the addon-bundle script instead.